### DISCUSSION TITLE
Error 403 / CAPTCHA?

### DISCUSSION THREAD

**Discussion by ghost** [jef/streetmerchant#1410] [Help]
After updating to the latest release of the code, for some reason I am starting to see new errors/warnings that I wouldn't have seen before.

I occasionally get "STATUS CODE ERROR 403" from **BestBuy** and I occasionally get "CAPTCHA" for **Amazon**.
The products that I am scanning for is the 5900x, Asus TUF RTX 3080, and the Asus TUF RTX 3080 OC.

I figured **BestBuy** would be an easy fix. Whenever I get a 403 error code from Newegg it means that I got banned, and need a new IP to continue scanning the website. So I changed my IP, made sure my IP was changed, but for some reason BestBuy keeps giving me the "STATUS CODE ERROR 403" messages randomly. It is like every 5th scan there will be an error code.

With **Amazon**... honestly I have no idea what to do or how to fix this, I figured the "CAPTCHA" error will go away eventually, and it hasn't still after I changed my IP numerous of times. I am mostly getting the "CAPTCHA" error with the Asus TUF RTX 3080.

What is going on? How do I fix this from happening? etc.

**Comment by FlyingToto** [comment]
duplicate of: [https://github.com/jef/streetmerchant/discussions/1298](url)
(although I don't get 403 from bestbuy)

**Reply by ghost** [reply]
I can't find how exactly to solve the "CAPTCHA" problem with Amazon after reading that article/post

**Reply by FlyingToto** [reply]
for the captcha to go away, try to run with: HEADLESS=false

**Comment by mikeike951** [comment]
I just tried to go to the article you linked but I just get a 404 error, stating it's not there.

**Reply by FlyingToto** [reply]
yeah for some reasons pasting the link doesn't seem to work either... disregard, 

for the captcha to go away, try to run with: HEADLESS=false
